cookbooks/lib/features/postgresql_spec.rb

```diff
@@ -8,11 +8,6 @@
     its(:exit_status) { should eq 0 }
   end
 
-  describe pgcommand('pg_config --bindir') do
-    its(:stdout) { should match(%r{/usr/lib/postgresql/9\.[2-6]/bin}) }
```

Comment:
Can the match be changed?  Maybe something like:

``` ruby
    its(:stdout) { should match(%r{/usr/lib/postgresql/(10\.|9\.[2-6])/bin}) }
```

Reply:
Sure, I just didn't see the value in the test unless we're specifically pinning the version. 
Also, it will break again when the next version is released.

Reply:
To me, this check is mostly to make sure the distro-specific postgresql version isn't the one in use, so maybe instead of a positive match we should be doing a negative match that asserts it *is not* postgresql 9.1 or earlier.

Reply:
Ah, that makes sense - will update for the negative match then. Thank you! 😄

Reply:
@meatballhat Hmm, on a second look, the Trusty-specific postgresql version is 9.3, which we already support.
We also already check the specific postgres version in the test just above this one, so this would just be a libpq-dev version check basically. 
So, I'm kind of leaning back towards my initial proposal of just getting rid of this test. 😞